asn1acn.cpp

```diff
@@ -109,6 +119,29 @@ ExtensionSystem::IPlugin::ShutdownFlag Asn1AcnPlugin::aboutToShutdown()
     return SynchronousShutdown;
 }
 
+void Asn1AcnPlugin::loadSettings()
```

Comment:
so serviceprovider will not use QSettings but some "asn1asc settings" class, which will be loaded here?